package.json

```diff
@@ -10,6 +10,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.1",
```

Comment:
just one question why 1.9.1 and not 1.9.2 ?

Reply:
New version just cam out today, upgraded to latest.